usage.md

```diff
@@ -2,15 +2,27 @@
 
 ## SYNOPSIS
 
-`tcurl` [--help] [-v | --version] [-H] [-p] [-t]
-        [-2 | --arg2 | --head] [-3 | --arg3 | --body]
-        [--shardKey] [--no-strict] [--timeout]
-        [--http] [--raw] [--health]
+`tcurl` <service> <endpoint> <options>
```

Comment:
```
`tcurl [OPTIONS] service endpoint
```

The options are usually not enumerated in the synopsis of a man page, and typically are for usage on the command line.